.contributing/the-process.md

```diff
@@ -167,15 +167,15 @@ Key
 2. "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* < User Flow Pieces < *Section* (CL)"
 
 3. "input task": 
-    - e.g Create a task in Asana with title "IS [1]: *title of Google file*" and:
-      - Add a link to the google drawing in the task description
```

Comment:
Maybe Google Doc instead? We could be getting input on use cases

Reply:
This is supposed to reference the new part of the file that says google drawing

Reply:
yess!!